reana-ui/src/pages/ReanaPage.js

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Would `MainPage.js/BasePage.js/[insert your suggestion]` instead of `ReanaPage.js` work as well? in case we want to keep it, then it would be `REANAPage.js`.

Reply:
`BasePage.js` sounds good IMO.